README.md

```diff
@@ -22,7 +22,6 @@ Install Bower dependencies
 ```bash
 $ polymer install
 ```
-
```

Comment:
Why remove this line?

Reply:
sorry remove it by wrong ,this is great work form grea egyption